File: sentimental-mario-less/mario.py
# Program to print a pyrmid similar to what is seen in Super Mario
while True:

    # Prompt user for pyramid height
    try:
        height = input("Height: ")
        height = int(height)
        if 1 <= height <= 8:
            break
        else:
            print("Height must be between 1 and 8.")
    except ValueError:
        print("Invalid usage.")

width = height

# Print right-aligned pyramid
for height in range(1, height + 1):
    for width in range(1, width):
        print(" ", end="")
    for column in range(0, height):
        print("#", end="")
    print("")

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

// Print a pyrmid similar to what is seen in Super Mario

int main(void)
{
    // Prompt user for height > 0
    // h = height, r = row, c = column, s = space
    int h;
    int r;
    int c;
    int s;
    do
    {
        h = get_int("Height:\n");
    }
    while (h < 1 || h > 8);

    // Print right-aligned pyramid
    for (r = 0; r < h; r++)
    {
        for (s = h - 1; s > r; s--)
        {
            printf(" ");
        }
        for (c = 0; c <= r; c++)
        {
            printf("#");
        }
        printf("\n");
    }
}
